### PR TITLE
refactor: format code spacing

### PR DESCRIPTION
**Pull request by AbhigyaKrishna**
## Description
Reformat project and enforce a `.editorconfig`
Fixes: #44 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,126 +1,126 @@
 plugins {
-    id 'com.android.application'
```
Comment (Vishwajith-Shettigar) [inline app/build.gradle:2]: please keep one tab space here and don't change anywhere tab space. In issue i have mentioned to remove unnecessary spaces from files. eg :  1<lines spaces.

File: app/src/main/java/com/example/nextgen/home/HomeActivity.kt [hunk 2]
```diff
@@ -124,9 +124,11 @@ class HomeActivity : BaseActivity(), ChatSummaryClickListener, RouteToEditProfil
 
             }
           }
+
           TYPE.CALL_ENDED -> {
             binding.callNotificationLayout.visibility = View.GONE
           }
+
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/home/HomeActivity.kt:131]: No need line space here.

File: app/src/main/java/com/example/nextgen/home/HomeFragment.kt [hunk 3]
```diff
@@ -74,6 +74,7 @@ class HomeFragment : BaseFragment() {
         is ChatViewModel -> {
           BaseAdapter.ViewType.CHAT
         }
+
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/home/HomeFragment.kt:77]: No need line space here.

File: app/src/main/java/com/example/nextgen/home/HomeViewModel.kt [hunk 4]
```diff
@@ -8,25 +8,25 @@ import com.example.nextgen.viewmodel.ObservableViewModel
 
 class HomeViewModel(
   private val chatController: ChatController,
-  private val userId:String,
+  private val userId: String,
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/home/HomeViewModel.kt:11]: please refactor such changes.

File: app/src/main/java/com/example/nextgen/recyclerview/BaseAdapter.kt [hunk 5]
```diff
@@ -49,10 +49,10 @@ class BaseAdapter<T> : RecyclerView.Adapter<BaseAdapter.BaseViewHolder<T>>() {
   class BaseViewHolder<T> internal constructor(
     private val binding: ViewBinding,
     private val viewType: ViewType,
-    private val expression: (T,ViewType, ViewBinding) -> Unit,
+    private val expression: (T, ViewType, ViewBinding) -> Unit,
   ) : RecyclerView.ViewHolder(binding.root) {
     fun bind(item: T) {
-      expression(item,viewType,binding)
+      expression(item, viewType, binding)
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/recyclerview/BaseAdapter.kt:55]: here also.

File: app/src/main/java/com/example/nextgen/service/VideoCallService.kt [hunk 6]
```diff
@@ -20,21 +20,21 @@ import com.example.videocallapp.TYPE
...
+class VideoCallService : android.app.Service() {
 
   @Inject
   lateinit var webSocketManager: WebSocketManager
 
   @Inject
   lateinit var context: Context
 
-  private lateinit var userId:String
+  private lateinit var userId: String
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/service/VideoCallService.kt:31]: here also.

File: app/src/main/java/com/example/nextgen/service/VideoCallService.kt [hunk 7]
```diff
@@ -46,27 +46,29 @@ class VideoCallService:android.app.Service() {
     (applicationContext as MyApplication).appComponent.inject(this)
 
     webSocketManager.message.observeForever { messageModel ->
-      if (messageModel?.type == null )
+      if (messageModel?.type == null)
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/service/VideoCallService.kt:49]: same here

**Review by Vishwajith-Shettigar — COMMENTED** [review]
Please only remove 1< extra line spaces in files.

**Comment by AbhigyaKrishna** [comment]
Shouldn't all these be consistent?

I created a `.editorconfig` to make these spaces and line indentations consistent.

**Comment by Vishwajith-Shettigar** [comment]
@AbhigyaKrishna, I appreciate your efforts, but some of the changes you made seems highly unnecessary. I have left review comments PTAL.

**Comment by Vishwajith-Shettigar** [comment]
Thanks @AbhigyaKrishna, This LGTM.

**Pull request merged**